cmd/clusterctl/client/cluster/template.go

```diff
@@ -129,6 +130,14 @@ func (t *templateClient) GetFromURL(templateURL, targetNamespace string, skipTem
 }
 
 func (t *templateClient) getURLContent(templateURL string) ([]byte, error) {
+	if templateURL == "-" {
```

Comment:
what about documenting this option in the command help and into https://cluster-api.sigs.k8s.io/clusterctl/commands/generate-cluster.html#alternative-source-for-cluster-templates

Reply:
Thanks, I've updated the docs and added a small note in the help message of the flag.